### DISCUSSION TITLE
YouTube embed broken?

### DISCUSSION THREAD

**Discussion by writingpoli** [knowledgefutures/pubpub#1329] [Support]
I'm getting the universal "no" sign when I try to insert YouTube video in a Pub:

[Screenshot](https://i.imgur.com/nRXRV70.png)

**Comment by bryangw1** [comment]
Same. Embed via iFrame also doesn't work.

**Reply by kaciccone** [reply]
I am also unable to embed a YouTube video. Did anyone from KFG ever respond to this question?

**Comment by gabestein** [comment]
Hey all — we were unable to replicate in 2021, but it looks like there is now an issue with all third-party embeds that appears to be new. We'll take a look.

**Reply by gabestein** [reply]
Hello! We've just pushed a fix for the issue.

**Reply by kaciccone** [reply]
It is still not working for me. After pasting in the YouTube URL, I can't click on "Insert" to insert the video.

**Reply by gabestein** [reply]
Can you provide a link to the video you're attempting to embed?

**Reply by kaciccone** [reply]
Okay, I just got it to work. I first tried to embed using the "share" link (https://youtu.be/IGYzs9Rhl1w). That didn't work. But I was just able to embed using the "watch" link (https://www.youtube.com/watch?v=IGYzs9Rhl1w). Earlier neither link would work. Thanks for your help.

**Reply by gabestein** [reply]
Glad to hear it! Yes, unfortunately the share links don't work with youtube's oEmbed endpoint, only the full watch links.
